tests/test_browser.py

```diff
@@ -3369,3 +3369,13 @@ def test_asmfs_test_fcntl_open(self):
 
   def test_asmfs_relative_paths(self):
     self.btest('asmfs/relative_paths.cpp', expected='0', args=['-s', 'ASMFS=1', '-s', 'USE_PTHREADS=1', '-s', 'FETCH_DEBUG=1', '--proxy-to-worker'])
+
+  def test_pthread_locale(self):
+    for args in [
+        [],
+        # ['-s', 'USE_PTHREADS=1'],
```

Comment:
Perhaps just remove this line? No need to test a non-pooled case if the test needs sync thread creation.